Modules/IndividualAssessment/classes/Orgu/class.ilIndividualAssessmentOrguHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+class ilIndividualAssessmentOrguHelper
+{
+	/**
+	 * @var ilOrgUnitUserAssignmentQueries
+	 */
+	protected $org_unit_user_assignment;
+
+	public function __construct(ilOrgUnitUserAssignmentQueries $org_unit_user_assignment)
+	{
+		$this->org_unit_user_assignment = $org_unit_user_assignment;
+	}
+
+	/**
+	 * @return int[]
+	 */
+	public function getEmployeesForUserId(int $user_id): array
+	{
+		$positions = $this->org_unit_user_assignment->getPositionsOfUserId($user_id);
+
+		$employees = array();
+		foreach ($positions as $position) {
+			$tmp_employees = $this->org_unit_user_assignment->getUserIdsOfOrgUnitsOfUsersPosition(
+				$position->getId(),
+				$user_id
+			);
+
+			foreach ($tmp_employees as $employer) {
+
+				array_push($employees, (int)$employer);
+			}
+		}
+		return array_unique($employees);
+	}
```

Comment:
This seems to collect the people that are in the same orgunit as the user. But instead we need to find the positions that allow the user to see other people in the IAss and for these find the people over which the positions give him authority. Please rename `getEmployeesForUserId` accordingly and fix the algorithm. Isn't there a query like this in the OrgUnits already? This method here will give to much people (e.g. the user himself) and also does not respect the permissions set for the positions at the IAss.